sql/updates/world/2018_04_18_02_world.sql

```diff
@@ -0,0 +1,5 @@
+-- Quest 28102 - Fight the power / Add Shadowstalkers to kill count
+UPDATE creature_template SET KillCredit1=7106 WHERE entry=7109;
+
+-- Quest 28460 - Threat of the Winterspring / Update creature kill credit
+UPDATE creature_template SET KillCredit1=48586 WHERE entry=7440 OR entry=7441 OR entry=7442 OR entry=10738;
```

Comment:
1.) so other can leave a comment, its best to add a empty line at end of the text..
2.) a other way to create a WHERE statement is:  WHERE entry IN (7440,7441,7442,10738);